src/ng/rootScope.js

```diff
@@ -418,8 +418,9 @@ function $RootScopeProvider() {
        *    expression in `watchExpressions` changes
        *    The `newValues` array contains the current values of the `watchExpressions`, with the indexes matching
        *    those of `watchExpression`
-       *    and the `oldValues` array contains the previous values of the `watchExpressions`, with the indexes matching
-       *    those of `watchExpression`
+       *    and the `oldValues` array contains the value of each `watchExpression` before the last change (it 
```

Comment:
I don't think this is clear enough. My head actually hurts a bit reading it. I will also see if I can come up with a better wording.